src/Time.coffee

```diff
@@ -1,14 +1,18 @@
-
 class CountDown
+
   constructor: ({@time = 120, @delay = 1000}) ->
     @onComplete = ->
     @onTick = ->
+    @pauseTimer = false
   start: (immediate = false) ->
     {delay, onTick, onComplete} = @
     interval = null
     tick = ->
-      onTick and onTick()
-      @time -= 1
+      # NOTE [scollins] well, at least this if statement and flag condition
```

Comment:
random `NOTE` comments shouldn't be committed. only `TODO` `FIXME` `BUG` should be included in commits (and functional comments when necessary)